src/Radar.js

```diff
@@ -0,0 +1,237 @@
+        .select(
+          elem("g.d3plus-Radar-radial-circles", {
+            parent: this._select,
+            enter: {transform},
+            update: {transform}
+          }).node()
+        )
+        .config(configPrep.bind(this)(this._shapeConfig, "shape", "Circle"))
+        .render()
+    );
+
+    const totalAxis = nestedAxisData.length;
+    const polarAxis = nestedAxisData
+      .map((d, i) => {
+        const angle = tau / totalAxis * i;
+        return {
+          id: d.key,
+          angle: 360 - 360 / totalAxis * i,
+          quadrant: parseInt(360 - 360 / totalAxis * i / 90, 10) % 4 + 1,
+          x: radius * Math.cos(angle),
+          y: radius * Math.sin(angle)
+        };
+      })
+      .sort((a, b) => a.key - b.key);
+
+    this._shapes.push(
+      new TextBox()
+        .data(polarAxis)
+        .rotate(
+          d => d.angle < 90 || d.angle > 270 ? -d.angle : -d.angle + 180
+        )
+        .rotateAnchor([0, 0])
+        .textAnchor(d => [1, 4].includes(d.quadrant) ? "start" : "end")
+        .height(14)
+        .text(d => d.id)
+        .x(d => d.x)
+        .y(d => d.y)
+        .select(
+          elem("g.d3plus-Radar-text", {
+            parent: this._select,
+            enter: {transform},
+            update: {transform}
+          }).node()
+        )
+        .render()
+    );
+
+    d3.selectAll("g.d3plus-Radar-text text")
+      .data(polarAxis)
+      .attr("x", d => d.x < 0 ? -10 : 10)
+      .attr("y", 5);
+
+    this._shapes.push(
+      new Path()
+        .data(polarAxis)
+        .d(d => `M${0},${0} ${-d.x},${-d.y}`)
+        .select(
+          elem("g.d3plus-Radar-axis", {
+            parent: this._select,
+            enter: {transform},
+            update: {transform}
+          }).node()
+        )
+        .config(this._shapeConfig.Circle)
+        .render()
+    );
+
+    const groupData = nestedGroupData.map(h => {
+      const q = h.values.map((d, i) => {
+        const angle = tau / totalAxis * i,
+              r = this._value(d, i) / maxValue * radius;
+        return {
+          x: r * Math.cos(angle),
+          y: r * Math.sin(angle)
+        };
+      });
+
+      const d = `M ${q[0].x} ${q[0].y} ${q
+        .map(l => `L ${l.x} ${l.y}`)
+        .join(" ")}`;
+
+      return {id: h.key, d};
+    });
+
+    this._shapes.push(
+      new Path()
+        .data(groupData)
+        .d(d => d.d)
+        .select(
+          elem("g.d3plus-Radar-items", {
+            parent: this._select,
+            enter: {transform},
+            update: {transform}
+          }).node()
+        )
+        .config(configPrep.bind(this)(this._shapeConfig, "shape", "Path"))
+        .render()
+    );
+
+    super._draw(callback);
+
+    return this;
+  }
+
+  /**
+      @memberof Radar
+      @desc If *value* is specified, sets the hover method to the specified function and returns the current class instance.
+      @param {Function} [*value*]
+      @chainable
+   */
+  hover(_) {
```

Comment:
this method does not do anything different from the default event inherited by `Viz`, please remove.